index/nrfconnect.json

```diff
@@ -171,6 +171,25 @@
           "sdk": "v2.3.0"
         }
       ]
+    },
+    {
+      "name": "asset-tracker-template",
+      "repo": "https://github.com/nrfconnect/Asset-Tracker-Template",
+      "title": "Asset Tracker Template",
+      "description": "Template for Cellular Asset Tracker applications for the nRF91 series SiPs.",
+      "kind": "template",
+      "tags": [],
+      "apps": "app",
+      "avatar": "https://avatars.githubusercontent.com/u/40860733?s=200&v=4",
+      "docsUrl": "https://github.com/nrfconnect/Asset-Tracker-Template/blob/main/README.md",
+      "releases": [
+        {
+          "date": "2024-12-17T16:52:00Z",
```

Comment:
@FilipZajdel Is there a way I can just specify the latest version here? Or do something that makes the code automatically pull in all the tagged versions? Its very inconvenient to update this for every release.

Reply:
ping 🚀

Reply:
My bad, we have discussed this before offline. Please ignore.